brave-intl/publishers
# Feat/3470 payout report status

## Pull Request Name

Add status tracking to payout report admin

#### Features

Calculate percent completion(may actually be boolean but *shrug*) add status indicators for different states.

#### How To Test

1. Login as an admin and generate a payout report
2. You will see two new columns in the index view, show and status
3. When a job succeeds it should show a complete status and completion percentage.  If a job fails it will report Error with the error message and 0% completion status.

#### Pull Request Checklist
